provider/cmd/pulumi-resource-command/schema.json

```diff
@@ -469,22 +469,22 @@
         "connection"
       ]
     },
-    "command:remote:CopyFile": {
```

Comment:
Should we include an alias for this rename? Probably not too significant because this resource doesn't really use state in any meaningful way.

Reply:
I need to do some testing around this. It's not only the name of the resource that changes, the main input property for the source of the copy changes as well. I think a replace might be unavoidable.

Reply:
I think you'll be okay as that property doesn't cause replacements.